### PR TITLE
New sample - SPFx React search with refiners

### PR DESCRIPTION
**Pull request by FranckyC**
| Q               | A
| --------------- | ---
| New sample?      | yes

#### What's in this Pull Request?

This sample shows how to use the SharPoint search REST API with refiners and paging using the _sp-pnp-js_ library and Office UI fabric components. This Web part is is fully generic and can be used without any additional customizations.

### CODE REVIEW

**Comment by FranckyC** [comment]
Hi @VesaJuvonen, any update on this?

**Comment by VesaJuvonen** [comment]
Sorry Franck, was planning to get the queue cleaned today - you sharing this as separate PR or?

**Comment by VesaJuvonen** [comment]
Or let me rephrase it - planning to submit updated on or? Just to ensure that we get it listed and processed. Looks really good and would be great to get it included here.

**Comment by FranckyC** [comment]
@VesaJuvonen Please wait with this PR. This sample is on hold. I'll let you know as soon as possible.

**Comment by FranckyC** [comment]
@VesaJuvonen Other samples are on hold too (react-collapsible-textboxio and react-textboxio).

**Comment by VesaJuvonen** [comment]
thx for the fast reply Francky, appreciated.

**Comment by FranckyC** [comment]
@VesaJuvonen This sample is on hold for now. I'll resubmit it in couple of weeks probably. However, I have an other sample to submit (react-textboxio). See PR.
